spec/static_collection_spec.rb

```diff
@@ -1,5 +1,34 @@
+require 'git'
+
 describe StaticCollection do
-  it "has a version number" do
-    expect(StaticCollection::VERSION).not_to be_nil
+  def get_version(git, branch = 'HEAD')
```

Comment:
I have no context here, but seems like you could include `ws-gem_publisher` and use the specs from there https://github.com/wealthsimple/ws-gem_publisher/blob/main/lib/ws/gem_publisher/support/version_rspec.rb#L58-L83?

Reply:
oh, i didn't know about that. Thanks

Reply:
On a separate note, maybe we can change how we release the gem here https://github.com/wealthsimple/static_collection/blob/main/.github/workflows/main.yml#L39-L52

Reply:
oh, also i just noticed this is a `public` gem 🙈  so please disregard my previous comment!